services/static-webserver/client/source/class/osparc/data/model/NodeProgressSequence.js

```diff
@@ -18,7 +18,7 @@
 /**
  * The progress sequence of a dynamic service is as follows:
  * - CLUSTER_UP_SCALING      (1)
- * - SIDECAR_PULLING         (2)
+ * - SIDECARS_PULLING        (2)
```

Comment:
why sidecars plural?

Reply:
actually, the proxy is also a sidecar, as is the envoy or whatever is running alongside the user services.